cmd/templatizer/app/flags.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2020 The Jaeger Authors.
```

Comment:
It's ok to not do this, but the name `templatizer` is very generic, I thought it would be some generic binary to apply templates. Since this is actually very specific to ES due to the flags supported by this binary, I would rename the package to something like `esmapping`

Reply:
makes sense. Will do

Reply:
esmapping-generator may be?